subprocess32.py

```diff
@@ -291,7 +291,7 @@ def call(*popenargs, **kwargs):
     p = Popen(*popenargs, **kwargs)
     try:
         return p.wait(timeout=timeout)
-    except TimeoutExpired:
+    except:
```

Comment:
Looks good, upstream fix was:

commit c15c88c13dd67e320161760638778b17486100c1
Author: Victor Stinner <victor.stinner@haypocalc.com>
Date:   Thu Sep 1 23:45:04 2011 +0200

    Issue #12494: Close pipes and kill process on error in subprocess functions
    
    On error, call(), check_call(), check_output() and getstatusoutput() functions
    of the subprocess module now kill the process, read its status (to avoid
    zombis) and close pipes.

I think that python choice of killing the process on timeout is bad, but it is too late to fix that now.

Also kill() will not work if you started a process using sudo, and in this case may raise and leave a zombie, but this should be fixed in upstream.

Reply:
The upstream 3.3+ commit did more, it also modified check_call().
  https://github.com/python/cpython/commit/c15c88c13dd67e320161760638778b17486100c1

My gut feeling is that this is a behavior difference we should just live with rather than backport into subprocess32.  The goal of this backport was reliability, not matching all of the 3.x changes.  (a bit muddled now that we have timeout support and the new run API... but those are additional APIs rather than behavior changes)